app/views/admin/infobase/_registered_users.html.haml

```diff
@@ -0,0 +1,14 @@
+%table.ui.very.basic.table{style: 'border: none'}
+  %thead
+    %tr
+      %th Name
+      %th E-mail
+      %th Date of Registration
+      %th.right.aligned Status
+  %tbody
+    - @users.each do |user|
+      %tr
+        %td{"data-label" => "Name"}= user.name
```

Comment:
`{ data: { label: ...`
I won't leave comments about it anymore, but please change everywhere. Current syntax kinda burns everybody's eyes